src/scheduled-tasks/messages/activismMon.ts

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+/*
+    Animal Rights Advocates Discord Bot
+    Copyright (C) 2022  Anthony Berg
```

Comment:
You should update the copyright to include the year and name, e.g. `Copyright (C) 2022, 2025   Anthony Berg, ...`